demo/src/app/pages/modules/datepicker/datepicker.page.ts

```diff
@@ -112,6 +119,13 @@ export class DatepickerPage {
                     defaultValue: "0"
                 },
                 {
+                    name: "pickerInitialView",
+                    type: "CalendarViewType",
+                    description: "Specifies the initial view for the datepicker. Options are: <code>CalendarViewType.Year</code>, " +
```

Comment:
Ahh I see, sorry completely forgot the fact that it was just an enum! Turning it into a string enum is definitely the way foward, as then it can be easily used in HTML